### ISSUE TITLE
Cant use

### ISSUE THREAD

**Issue by marvinto23** [JulianPrieber/polygon#1]
I get an Server Error 500 if i select the Theme and apply and then open the site

**Comment by JulianPrieber** [comment]
Does this only happen with this theme?

**Comment by marvinto23** [comment]
Yes, i also tryed it on diffrent Installation its just with this Theme

**Comment by JulianPrieber** [comment]
_The good news is I couldn't reproduce this on any of my systems.
The bad news is I couldn't reproduce this on any of my systems._

This doesn't appear to be something that is caused by the theme itself.

The error you're describing is server side, so this has to be something wrong with your setup or the instance itself.

To be sure, I'd recommend you turn on debugging in the config, this should turn the error into some error code we can work with.

On the diagnosis tab on the config page, please make sure that you have all the dependencies you need.

Also, make sure you're using the latest version of LittleLink Custom and the theme.

If nothing helps I'd suggest looking into the docker version as this offers a controlled environment where everything is optimized to run everything properly.

**Comment by marvinto23** [comment]
Hello

I attached the screenshots
Thanks
![388D591E-8BFC-4BEC-9930-912172D92699](https://user-images.githubusercontent.com/84294162/212891560-7c916b9e-ff9d-4f42-a8ff-a4cba2292b01.png)
![53F7DEE4-9BEE-4CF4-94B0-F827742B1EAD](https://user-images.githubusercontent.com/84294162/212891571-ad7e3d32-6b04-4c6c-8e0d-a1d0cbd70279.png)

**Comment by JulianPrieber** [comment]
How have you uploaded the theme?
Have you ever uploaded other themes as well?

**Comment by Sev7nrayne** [comment]
@marvinto23 when you unzip the files remove the version from the folder name.
themes\polygon-1.3 -> themes\polygon
